src/types/apiNodeTypes.ts

```diff
@@ -2,3 +2,14 @@ export interface ApiNodeCost {
   name: string
   cost: number
 }
+
+export interface ApiNodeCostData {
```

Comment:
Shall we document each field here?

nit: Do we really need `vendor` field if in the real data, the field is always `Unknown`?